pages/en/lb3/Making-authenticated-requests.md

```diff
@@ -36,13 +36,15 @@ http://localhost:3000/api/widgets
 curl -X GET http://localhost:3000/api/widgets?access_token=$ACCESS_TOKEN
 ```
 
-To use cookies for authentication, add the following to server.js (before boot):
-
-{% include code-caption.html content="/server/server.js" %}
-```javascript
-app.use(loopback.token({
-    model: app.models.accessToken
-}));
+To use cookies for authentication, add the following to middleware.json :
```

Comment:
Can we preserve both the programatic and declarative way of doing this?

Reply:
you're right I add the 2 options (and replace app.use by app.middleware for phases)